var/spack/repos/builtin/packages/fdupes/package.py

```diff
@@ -20,6 +20,7 @@ class Fdupes(AutotoolsPackage):
     variant('ncurses', default=True, description='ncurses support')
 
     depends_on('ncurses', when='+ncurses')
+    depends_on('pcre2', type='link')
```

Comment:
Sorry I missed this dependency initially. :disappointed:

According to https://github.com/adrianlopezroche/fdupes/blob/master/INSTALL.enduser, pcre2 seems to be only required when building with ncurses support, so we should make this:
```suggestion
    depends_on('pcre2', when='+ncurses')
```

Reply:
I fixed it as pointed out.

I confirmed that the following 2 patterns can be built on aarch64 machine.
$ spack install fdupes %gcc@8.3.1
$ spack install fdupes~ncurses %gcc@8.3.1